routes/mapstats.js

```diff
@@ -93,15 +179,15 @@ router.post("/create", Utils.ensureAuthenticated, async (req, res, next) => {
       !Utils.superAdminCheck(req.user)
     ) {
       res
-        .status(401)
+        .status(403)
         .json({ message: "User is not authorized to perform action." });
       return;
     } else if (
       matchRow[0].cancelled == 1 ||
       matchRow[0].forfeit == 1 ||
       matchRow[0].end_time != null
     ) {
-      res.status(401).json({ message: "Match is already finished." });
+      res.status(422).json({ message: "Match is already finished." });
```

Comment:
Same as above.

Reply:
Some tests in mapstats are still failing. I think there's some legacy stuff leftover from the `update` and `create` I think!

Reply:
Made a small PR to update the test cases in mapstats. Ran the tests on my end and they seem to be working now. The issue with Travis is that there are two secrets (database key, and Steam API key) so the merging tests will always fail... I need to think of a way to mock steam resolver requests. But if you merge in those changes we should be able to merge in this branch whenever you're ready!